core/app/models/workarea/order.rb

```diff
@@ -242,29 +242,35 @@ def place
     def add_item(attributes)
       quantity = attributes.fetch(:quantity, 1).to_i
       sku = attributes[:sku]
-      customizations = attributes[:customizations]
 
-      if existing_item = items.find_existing(sku, customizations)
-        update_item(existing_item.id, quantity: existing_item.quantity + quantity)
+      if existing_item = items.find_existing(sku, attributes)
+        attributes[:quantity] = existing_item.quantity + quantity
+
+        update_item(existing_item.id, attributes.except(:via))
```

Comment:
still think quantity should be the only thing pass through here.

Reply:
we did talk about that, yeah. dunno what i was thinking.